client/state/selectors/get-site-options.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import { getRawSite } from 'state/sites/selectors';
+
+/**
+ * Returns the site options
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}    siteId   Site ID
+ * @returns  {?Object}   options  Site options or null
+ */
+export default ( state, siteId ) => {
+	const site = getRawSite( state, siteId );
+	if ( ! site ) {
+		return null;
+	}
+	const options = site.options ? { ...site.options } : {};
```

Comment:
- When is `site.options` falsey?
- I think we should avoid creating a shallow clone of options if we don't always mutate it. In other words, move the object spread into the condition below:

```js
let options = site.options || {};
const defaultPostFormat = options.default_post_format;
if ( ! defaultPostFormat || defaultPostFormat === '0' ) {
	options = { ...options, default_post_format: 'standard' };
}

return options;
```